clang/tools/libclang/CIndexer.cpp

```diff
@@ -17,6 +17,7 @@
 #include "clang/Driver/Driver.h"
 #include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/SmallString.h"
+#include "llvm/Config/config.h"
```

Comment:
This is a private LLVM header, see the comment at the top of the file. It cannot be included in other subprojects.

You either have to export this macro from llvm-config.h, or you have to add it to clang's config.h.

Reply:
Thanks, I created a fix for it here https://github.com/llvm/llvm-project/pull/75928